velox/experimental/cudf/exec/CudfHashJoin.cpp

```diff
@@ -146,7 +161,15 @@ void CudfHashJoinBuild::noMoreInput() {
   };
 
   auto stream = cudfGlobalStreamPool().get_stream();
-  auto tbl = getConcatenatedTable(inputs_, stream);
+  std::unique_ptr<cudf::table> tbl;
+  if (inputs_.size() == 0) {
```

Comment:
Should we move this empty inputs logic to getConcatenatedTable, and may also refactor otherwhere
```
std::unique_ptr<cudf::table> makeEmptyTable(TypePtr const& inputType) {
  std::vector<std::unique_ptr<cudf::column>> emptyColumns;
  for (size_t i = 0; i < inputType->size(); ++i) {
    auto emptyColumn = cudf::make_empty_column(
        cudf_velox::veloxToCudfTypeId(inputType->childAt(i)));
    emptyColumns.push_back(std::move(emptyColumn));
  }
  return std::make_unique<cudf::table>(std::move(emptyColumns));
}
```